packages/wrangler/src/publish/publish.ts

```diff
@@ -828,19 +827,7 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 		logger.log("No publish targets for", workerName, formatTime(deployMs));
 	}
 
-	try {
-		const deploymentsList = await fetchResult<DeploymentListResult>(
-			`/accounts/${accountId}/workers/deployments/by-script/${scriptTag}`
-		);
-
-		logger.log("Current Deployment ID:", deploymentsList.latest.id);
-	} catch (e) {
-		if ((e as { code: number }).code === 10023) {
-			// TODO: remove this try/catch once deployments is completely rolled out
-		} else {
-			throw e;
-		}
-	}
+	logger.log("Current Deployment ID:", deploymentId);
```

Comment:
This will cause the deployment ID to not hyphens, I'm not sure if that's unpolished/confused to users

Reply:
Same [here](https://github.com/cloudflare/workers-sdk/pull/2607/files#diff-605d740517c44d6d11fe53bd4718c64bf8baa876c6f1af516e0aa5efcfe3c977R149)